lib/invalid_engine_error.js

```diff
@@ -0,0 +1,27 @@
+var util = require('util')
+
+function InvalidEngineError (currentEngine, specifiedEngines) {
+  // Super constructor
+  Error.call(this)
+
+  // Super helper method to include stack trace in error object
+  Error.captureStackTrace(this, this.constructor)
+
+  if (specifiedEngines && Object.keys(specifiedEngines).length > 0) {
+    this.message = [
+      'Current engine: ' + currentEngine + ' is not specified in',
+      'package.json. Specified engines are:',
+      Object.keys(specifiedEngines).join(', ')
+    ].join(' ')
+  } else {
+    this.message = [
+      'No engines specified. Please add an engine to your package.json'
+    ].join(' ')
```

Comment:
[invalid_dependency_error.js#L3](https://github.com/westfieldlabs/environment_validator/blob/version_1/lib/invalid_dependency_error.js#L3) and [invalid_engine_version_error.js#L3](https://github.com/westfieldlabs/environment_validator/blob/version_1/lib/invalid_engine_version_error.js#L3)  also have lines over 80 characters in case you wanted to chop those up as well.  You might want to add eslint and use the [max-len](http://eslint.org/docs/rules/max-len.html) rule to automatically check for this stuff.